nwbwidgets/misc.py

```diff
@@ -287,10 +289,10 @@ def show_psth(units: pynwb.misc.Units, index=0, start_label='start_time', before
             labels, cvals = np.unique(coldata, return_inverse=True)
         elif np.all(np.isreal(data)):
             coltype = 'numeric'
-            cvals = coldata
+            labels, cvals = np.unique(coldata, return_inverse=True)
```

Comment:
`labels` is only ever used if `coltype == 'cat'`, so I don't think this is necessary. What was the intention behind adding it?